.github/workflows/jscpd.json

```diff
@@ -1,7 +1,9 @@
 {
     "pattern": "packages/**/*.ts",
-    "ignore": ["**node_modules**", "**dist**"],
+    "ignore": ["**node_modules**", "**dist**", "*/scripts/*"],
```

Comment:
I wonder if this should be `**/scripts/**`

Reply:
Yep, verified you are correct. Here's the followup: https://github.com/aws/aws-toolkit-vscode/pull/5930